app/views/admin/registrations/index.html.haml

```diff
@@ -44,24 +44,14 @@
               %td
                 = link_to 'Questions','#', class: 'btn btn-success question-btn', 'data-id' => index, 'data-name' => registration.name
             %td
+              = check_box_tag @conference.short_title, registration.id, true, class: 'switch-checkbox', method: :patch,
+                data: { size: "large",
```

Comment:
The visually wrong alignment is caused by the line-height of the div. We might want to look into explicitely defining it in our css? What do you think @ChrisBr ?

+1 on the additional column OR just include it to the .btn-group, I think logically it makes sense to have it there.